src/ApiRequests.jsx

```diff
@@ -1,45 +1,84 @@
 import axios from "axios";
 
 const getArticles = () => {
-    return axios
-      .get("https://abcodesnorthcodersncnewsapp.onrender.com/api/articles")
-      .then(({ data }) => {
-      return data.articles
-      });
-  }
+  return axios
+    .get("https://abcodesnorthcodersncnewsapp.onrender.com/api/articles")
+    .then(({ data }) => {
+      return data.articles;
+    });
+};
 
-const getArticleByID = ({articleId}) => {
+const getArticlesByTopic = (topic) => {
```

Comment:
Rather than having a separate function here, we can just amend the getArticles api call by using axios params. This makes things a bit easier as it means you don't have to have a separate component (which you currently have as ArticlesByTopic) 👍